### PR TITLE
refactor: add `uvloop` to container dependencies

### PR DESCRIPTION
**Pull request by RogerHYang**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -111,6 +111,7 @@ container = [
   "opentelemetry-instrumentation-grpc",
   "py-grpc-prometheus",
   "strawberry-graphql[opentelemetry]==0.227.2",  # need to pin version because we're monkey-patching
+  "uvloop; platform_system != 'Windows'",
```
Comment (mikeldking) [inline pyproject.toml:114]: What bug did you find?

**Review by mikeldking — APPROVED** [review]

**Comment by axiomofjoy** [comment]
We just install it, we don't need to do any configuration in code?

**Comment by RogerHYang** [comment]
> We just install it, we don't need to do any configuration in code?

No. See [here](https://github.com/encode/uvicorn/blob/0efd3835da6dcc713f74aadf7b52779d0d1fa17d/uvicorn/loops/auto.py#L3).

**Comment by axiomofjoy** [comment]
@anticorrelator Just wanted to make sure you saw this in case it has any ramifications for evals. It's a subtle change.

**Pull request merged**